### ISSUE TITLE
Add a window to choose interface language for the installer

### ISSUE THREAD

**Issue by wx00** [FastCopyLab/FastCopy#254]
Hi, I'm using a Chinese system, and the new installer seem to follow system language by default. 
But the words are too different from other software, I can't understand some options.
I think it would be nice to have a language option window pop up at the very first of the installing process.
Thank you for maintaining and improving fastcopy.

**Comment by shirouzu** [comment]
> But the words are too different from other software, I can't understand some options.

Can you tell me a few example?

**Comment by wx00** [comment]
This sentence in EULA first line doesn't look like a Chinese sentence to me, although basically nobody cares about EULA when installing.
"（必须Pro许可证以便在工作场所使用等。然而，不包括为期两周的试用期）"

In the next window, I don't remember i've seen any usage of "另见" or "在一个方案组注册" in other software.
"另见" is in the "where to install" section so I can figure out from context, "在一个方案组注册" is weird, I know it's registering to a "方案组", but when I search this word it doesn't even connect to computer.

**Comment by shirouzu** [comment]
> This sentence in EULA first line doesn't look like a Chinese sentence to me

Sorry, I would like to make Chinese sound more natural.
https://github.com/FastCopyLab/FastCopy/issues/202

> although basically nobody cares about EULA when installing.

This is a contract clause that users must be aware of before starting use.

> "另见" or "在一个方案组注册"

Hum... "另见" intends "Browse (Browse installation directory)", and "在一个方案组注册" intends "Register to Program Group".
Do you think "浏览" is better than "另见"?

> "在一个方案组注册" is weird, I know it's registering to a "方案组", but when I search this word it doesn't even connect to computer.

Do you think "在一个程序组注册" is better than "在一个方案组注册"?
